docs/getting-started/quickstart.md

```diff
@@ -155,11 +154,8 @@ You should see similar output in your terminal, with your own randomly generated
 ## Step 4: Choose a remote infrastructure location
 
 Let's get this workflow running on infrastructure other than your local machine!
-We can tell Prefect where we want to run our workflow by creating a [work pool](/concepts/work-pools/).
-
-We can have Prefect Cloud run our flow code for us with a Prefect Managed work pool.
-
-Let's create a [Prefect Managed work pool](/guides/managed-execution/) so that Prefect can run our flows for us.
+Prefect refers to infrastructure that can run worflows as [work pools](/concepts/work-pools/). 
+Prefect can orchestrate workflows that run remotely on many [types of infrastructure](/work-pools/#work-pool-types), but here we'll use a [Prefect Managed work pool](/guides/managed-execution/) so that Prefect can run this flow for us.
```

Comment:
```suggestion
Prefect can orchestrate workflows that run remotely on many [types of infrastructure](/work-pools/#work-pool-types).  We'll use a [Prefect Managed work pool](/guides/managed-execution/) so that Prefect can run this flow for us.
```